docs/advanced-topics/client-anticipation.md

```diff
@@ -0,0 +1,117 @@
+
+In some cases, this may result in an undesirable player experience, causing things that normally move smoothly to become choppy. When this happens, you can smooth out the result by using the `Smooth()` function on both `AnticipatedNetworkVariable` and `AnticipatedNetworkTransform`.
+
+`Smooth()` takes a starting position and an ending position (which would usually be the prevoius anticipated value and either the new anticipated value or the current authoritatvie value) and a duration over which to perform the smoothing. For `AnticipatedNetworkVariable`, because the type is generic, it also requires a delegate to use to perform the smoothing operation (which takes a from value, a to value, and a percent value, and in many cases may just be `Mathf.Lerp`).
+
+Because the first parameter to `Smooth()` is usually the previous anticipated value, and by the time the `OnReanticipate` function is called, the anticipated value has been overwritten with the authoritative value, you can use `PreviousAnticipatedValue` on `AnticipatedNetworkVariable` and `PreviousAnticipatedState` on `AnticipatedNetworkTransform` to access the value you most recently passed into `Anticipate()` or its related transform methods. Note, however, that if you call `Anticipate()` or its related transform methods prior to doing the smoothing, they will update the previous anticipated value, so you may need to store the previous anticipated value in a local variable before doing your reanticipation.
+
+### Server-Side Smoothing
+
+Even though these concepts are mostly for client-side use, there are cases where you might want to perform server-side smoothing as well. For example, when running in host mode, the host player will see the movement of other players. Due to latency and jitter, input updates from remote clients may not arrive every frame, which may result in choppy movement of an AnticipatedNetworkTransform, for example.
+
+To address this, you can also use the `Smooth()` function at any time on the server side. The server does not do anticipation, so it will not get any `OnReanticipate()` events, but you can, for example, call `Smooth()` each time you receive an input from the client in order to smooth the motion between the previous position and the new one.
+
+One important distinction between client-side smoothing and server-side smoothing for `AnticipatedNetworkTransform`, though, is that client-side smoothing smooths the actual motion of the transform, while server-side smoothing only smooths the visual depiction of the motion. Which is to say, `AnticipatedState` is updated, but `transform` is not, so all game logic and collision detection will be done based on the actual position, and the rendering of the object will be slightly behind the actual position of the object.
```

Comment:
As a user, how would I access the authoritative value vs the visual value? What APIs do I use? For example, if my camera should track the interpolated value, how do I access it?

Reply:
From going back above, I'm assuming that's gameObject.transform for interpolated? And this.GetComponent<AnticipatedNetworkTransform>().authoritativeState for authoritative value?